WcaOnRails/config/locales/en.yml

```diff
@@ -1216,7 +1216,7 @@ en:
       #context: WEC
       wec:
         name: "WCA Ethics Committee"
-        description: "This committee has the role to ensure that all the procedures followed by the WCA Staff and External Staff abide by its Code of Ethics."
+        description: "This committee has the role to ensure that all the procedures followed by the WCA Staff and External Staff abide by its Code of Ethics, and to perform independent investigations if the conduct of any of these members is not compliant with the said Code of Ethics. Such investigations can be requested by any Registered Speedcubers, for which the WEC is a confidant. This committee also function as the board of appeal if anyone involved disagree with a decision made by the WCA Disciplinary Committee."
```

Comment:
Hm... maybe "for whom" instead of "for which"? It's referring to the "Registered Speedcubers", right?

"This committee also function" -> "This committee also function**s**". Similarly "disagree" -> "disagree**s**".